Neuro/Recording/RecordingHandler.cs

```diff
@@ -33,23 +37,29 @@ public void FixedUpdate()
             PlayerControl.LocalPlayer.killTimer,
             NeuroPlugin.Instance.Movement.DirectionToNearestTask,
             PlayerControl.LocalPlayer.myTasks.ToArray().Any(PlayerTask.TaskIsEmergency),
-            Vector2.zero,
+            NeuroPlugin.Instance.Impostor.DirectionToNearestVent,
+            NeuroPlugin.Instance.Impostor.NearbyVents,
             NeuroPlugin.Instance.Vision.DirectionToNearestBody,
             GameManager.Instance.CanReportBodies() && HudManager.Instance.ReportButton.isActiveAndEnabled,
             NeuroPlugin.Instance.Vision.PlayerRecords.ToDictionary(kv => kv.Key.PlayerId, kv => kv.Value),
             NeuroPlugin.Instance.Movement.LastMoveDirection,
             DidReport,
             DidVent,
+            PlayerControl.LocalPlayer.inVent,
             DidKill,
-            // TODO: Implement these two
-            false,
-            false
+            DidSabotage,
+            SabotageUsed,
+            DidDoors,
+            NeuroPlugin.Instance.Impostor.NearbyDoors,
+            DoorsUsed
         );
         Frames.Add(frame);
 
         // string frameString = JsonSerializer.Serialize(frame);
         // Info(frameString);
 
-        DidReport = DidVent = DidKill = false;
+        DidReport = DidVent = DidKill = DidSabotage = DidDoors = false;
+        SabotageUsed = SabotageTypes.None;
+        DoorsUsed.Clear();
```

Comment:
Is it enough for this information to only be recorded in one single frame?

Reply:
@Vedal987 Your input on this please

Reply:
This is one element that currently concerns me in regards to training the AI, that being said the AI should only output this for one frame therefore at the moment I'm inclined to just record for one frame provided there is an input for whether its possible to do.

Reply:
I believe there's enough inputs right now to objectively decide whether to vent/sabotage. If we ever need to it shouldn't be very hard to modify this reset behavior based on certain game states.